src/collections/PublicationDetail.ts

```diff
@@ -0,0 +1,117 @@
+import { CollectionConfig } from 'payload';
+import { fieldsTabMeta } from '@/field-templates/meta';
+import { TextBlock } from '@/blocks/TextBlock';
+
+export const PublicationDetailConfig: CollectionConfig = {
+  access: {
+    read: (): boolean => true,
+  },
+  admin: {
+    group: 'Pages',
+  },
+  fields: [
+    {
+      admin: {
+        hidden: true,
+      },
+      defaultValue: true,
+      name: 'isLinkable',
+      type: 'checkbox',
+    },
+    {
+      tabs: [
+
+        // Content Tab
+        {
+          fields: [
+
+            // Overview Page properties
+            {
+              fields: [
+                {
+                  name: 'image',
+                  relationTo: 'images',
+                  required: true,
+                  type: 'relationship',
+                },
+              ],
+              label: 'Overview Page properties',
+              name: 'overviewPageProps',
+              type: 'group',
+            },
+
+            // Categorization
+            {
+              fields: [
+                {
+                  name: 'topic',
+                  relationTo: 'publicationTopics',
+                  required: true,
+                  type: 'relationship',
+                },
+                {
+                  name: 'type',
+                  relationTo: 'publicationTypes',
+                  required: true,
+                  type: 'relationship',
+                },
+              ],
+              label: 'Categorization',
+              name: 'categorization',
+              type: 'group',
+            },
+
+            // Hero
+            {
+              fields: [
+                {
+                  localized: true,
+                  name: 'title',
+                  required: true,
+                  type: 'text',
```

Comment:
For titles we should generally allow the simple rte config. Otherwise we can not have titles with soft hyphens, subscript and superscript.